README.md

```diff
@@ -3379,7 +3379,167 @@ Response:
 }
 ```
 
+## GET /api/workflows/:version/callcaching/diff
 
+This endpoint returns the hash differences between 2 *completed* (successfully or not) calls.
+The following query parameters are supported:
+
+| Parameter |                                        Description                                        | Required |
+|:---------:|:-----------------------------------------------------------------------------------------:|:--------:|
+| workflowA | Workflow ID of the first call                                                             | yes      |
+| callA     | Fully qualified name of the first call. **Including workflow name**. (see example below)  | yes      |
+| indexA    | Shard index of the first call                                                             | no       |
+| workflowB | Workflow ID of the second call                                                            | yes      |
+| callB     | Fully qualified name of the second call. **Including workflow name**. (see example below) | yes      |
+| indexB    | Shard index of the second call                                                            | no       |
+
+
+cURL:
+
+```
+$ curl "http://localhost:8000/api/workflows/v1/callcaching/diff?workflowA=85174842-4a44-4355-a3a9-3a711ce556f1&callA=wf_hello.hello&workflowB=7479f8a8-efa4-46e4-af0d-802addc66e5d&callB=wf_hello.hello"
+```
+
+HTTPie:
+
+```
+$ http "http://localhost:8000/api/workflows/v1/callcaching/diff?workflowA=85174842-4a44-4355-a3a9-3a711ce556f1&callA=wf_hello.hello&workflowB=7479f8a8-efa4-46e4-af0d-802addc66e5d&callB=wf_hello.hello"
+```
+
+Response:
+```
+HTTP/1.1 200 OK
+Content-Length: 1274
+Content-Type: application/json; charset=UTF-8
+Date: Tue, 06 Jun 2017 16:44:33 GMT
+Server: spray-can/1.3.3
+
+{
+  "callA": {
+    "workflowId": "85174842-4a44-4355-a3a9-3a711ce556f1",
+    "callFqn": "wf_hello.hello",
+    "jobIndex": -1,
+    "allowResultReuse": true
```

Comment:
Agreed that the call status would be very nice, the only problem is I don't think we can currently know that without querying metadata ?